.github/workflows/docker-build-publish.yaml

```diff
@@ -130,14 +130,15 @@ jobs:
 
       # The Github action runs CIS Dockerfile benchmark against dockerfiles in repository (CIS 4.1, 4.2, 4.3, 4.6, 4.7, 4.9, 4.10)
       # https://github.com/sysdiglabs/benchmark-dockerfile
+      # P.S. SKIP CIS 4.1! before https://github.com/yahoojapan/authorization-proxy/pull/95 fixed.
```

Comment:
```suggestion
      # TODO: Skipping CIS 4.1 check until https://github.com/yahoojapan/athenz-client-sidecar/pull/58 is fixed.
```

Reply:
fixed in [6bfd033](https://github.com/AthenZ/authorization-proxy/pull/2/commits/6bfd033bc3cc0ebf731c85a0e151795005b990c6)